### ISSUE TITLE
Request for new release

### ISSUE THREAD

**Issue by sallner** [lordmauve/flake8-html#17]
### Description

After #16 was merged it would be nice to have a release as otherwise these changes do not have any effect in CI.

@lordmauve  Thank you for the work!

**Comment by lordmauve** [comment]
Yes, I have already started doing it.

**Comment by lordmauve** [comment]
Released 0.4.1. It took more effort than I thought as I had to update some of the tox paraphernalia and compile myself a Python 3.8 on this host.

**Comment by sallner** [comment]
Thank you very much for your effort!
